packages/eas-cli/src/commands/secrets/delete.ts

```diff
@@ -0,0 +1,45 @@
+import { Command } from '@oclif/command';
+
+import { EnvironmentSecretMutation } from '../../graphql/mutations/EnvironmentSecretMutation';
+import Log from '../../log';
+import { toggleConfirmAsync } from '../../prompts';
+import { ensureLoggedInAsync } from '../../user/actions';
+
+export default class EnvironmentSecretDelete extends Command {
+  static hidden = true;
+  static description = `
+Delete an environment secret by ID.
+Unsure where to find the secret's ID? Run ${'`eas secrets:list`'}
+  `;
+
+  static args = [
+    {
+      name: 'id',
+      required: true,
+      description: `ID of the secret to delete`,
+    },
+  ];
+
+  async run() {
+    await ensureLoggedInAsync();
+
```

Comment:
Maybe we should check here if EAS is enabled/available for the project? The same way you're checking it in the other two commands.